### ISSUE TITLE
Accessibility - Every Input should have a label

### ISSUE THREAD

**Issue by adrianjost** [JohMun/vue-tags-input#100]
Hi, 
I have discovered another accessibility issue. It's a very small one.
Every input should always have an appropriate label. A simple fix would be to add
`aria-label="enter new tag"` at [line 130](https://github.com/JohMun/vue-tags-input/blob/master/vue-tags-input/vue-tags-input.vue#L130). You can read more about `aria-label`
[here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute).

A more complex (but better) fix would be to expose the `aria-label` attribute with a prop. This way it is possible to adapt it for localization and similar wording changes.
